app/grandchallenge/reader_studies/models.py

```diff
@@ -1040,6 +1040,7 @@ class QuestionWidgetKindChoices(models.TextChoices):
 
 
 ANSWER_TYPE_TO_QUESTION_WIDGET = {
+    "": [],  # Blank answer type
```

Comment:
Why is this necessary? If answer type is left blank, it will be `None` and we return `ComponentInterface.objects.none()`, right?

Reply:
I could have sworn this was to resolve a bug I discovered updating from one answer type to the next. However, I cannot reproduce it.

I'll just remove it.

Also: this resolves around widget types not CIs =)